SvelteCompiler.js

```diff
@@ -1,6 +1,34 @@
-import htmlparser from 'htmlparser2';
-import sourcemap from 'source-map';
-import svelte from 'svelte/compiler';
+import { readFileSync } from 'fs';
+import findUp from 'find-up';
+
+// Read package information from user's `package.json`
+const pkgPath = findUp.sync('package.json');
+const pkg = pkgPath ? JSON.parse(readFileSync(pkgPath, 'utf8')) : {};
+const packages = Object.assign({}, pkg.dependencies, pkg.devDependencies);
+const root = pkgPath.substring(0, pkgPath.length - 12);
+
+// prefer the user installed/managed version
+function req (name) {
+  let mod;
+  if (pkgPath && packages[name]) {
+    try {
+      mod = require(root + 'node_modules/' + name);
```

Comment:
`root` is an absolute file system path and Meteor's `require` resolves absolute paths relative to the project's root directory, so this will always result in a `Cannot find module` error.

Reply:
This actually worked on Windows. It'd probably easy enough to resolve the path, but it sounds like this is all undeeded.